trame_iframe/widgets/iframe.py

```diff
@@ -117,12 +119,18 @@ def __init__(self, event_names=[], **kwargs):
         Communicator._next_id += 1
         ref_name = f"trame_comm_{Communicator._next_id}"
         self.__ref = kwargs.get("ref", ref_name)
+        self._parent_origin = parent_origin
 
         super().__init__(
             "communicator",
             **kwargs,
         )
+        self._attr_names += [
+            ["parent_origin", "parentOrigin"],
+        ]
         self._attributes["ref"] = f'ref="{self.__ref}"'
+        self._attributes["parent_origin"] = f'parentOrigin="{self._parent_origin}"'
```

Comment:
no need to set _attributes if you add it to ` self._attr_names`

Reply:
one or the other but not both

Reply:
Understood, just cleaned up